template.md

```diff
@@ -0,0 +1,40 @@
+# <SOMETHING> means "change me"
+# FIELD? means "optional"
+
+---
+title: "<TITLE>"
+description: "<DESCRIPTION>"
+authors:
+  - <AUTHOR1>
+  - <AUTHOR2>
+  - <AUTHORN>
+links:
+  gh?: "https://github.com/<USER>/<PROJECT>"
```

Comment:
I think the format should be `<USER>/<PROJECT>` (without the preceding URL). That is the purpose of having `gh:`, `gl:`, `bb:`, etc.

Alternatively, we might make "repos" a list and let any URL:

```yml
links:
  repos:
    - <URL1>
    - <URL2>
```

However, this would be harder to later show custom icons depending on the service.